pkg/arvo/sys/zuse.hoon

```diff
@@ -5538,9 +5538,9 @@
         ==
         ;~  pose
           ;~  pfix
-            (just 'e')
+            ;~(pose (just 'e') (just 'E'))
```

Comment:
`+just` is deprecated in favor of `+jest` I believe. But also, consider using `+mask`.

```suggestion
            (mask "eE")
```